ui/components/src/TimeSeriesTooltip/tooltip-model.ts

```diff
@@ -59,6 +59,10 @@ export interface CursorCoordinates {
     x: number;
     y: number;
   };
+  page: {
+    x: number;
+    y: number;
+  };
```

Comment:
Do we still need the `viewport` data (answer might be yes, I don't know a ton about this area of the code)

Reply:
yeah good catch, I was thinking we may possibly need it later but coming back to this, I'm not really sure when we ever would so went ahead and removed it